lib/checks/visibility/hidden-content.js

```diff
@@ -1,6 +1,8 @@
 let styles = window.getComputedStyle(node);
+let nodeName = node.nodeName.toUpperCase();
+let skipOver = ['HEAD', 'TEMPLATE', 'SCRIPT'];
```

Comment:
I don't think noscript should be there. That's one of those elements that could actually become visible under certain conditions.

`style` on the other hand should be on this list.